src/main/java/com/tourguide/myclassjournal/TourGuideApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.tourguide.myclassjournal;
+
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TourGuideApplication {
+    
+    public static void main(String[] args) {
+        SpringApplication.run(TourGuideApplication.class, args);
+    }
+}
```

Comment:
I saw that the application and controller for the project wasn't around. I checked the commits and sure enough, they had been [deleted](https://github.com/bolandmj/TourGuide/commit/f34b5e9f132c256e994b294e298ad83383329e18). I don't know why that is, but I do know that it's important to at least have an application. So I created a simple TourGuideApplication
Source: [https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3](https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3)